templates/nextjs/src/hooks/useBrowserWallet.tsx

```diff
@@ -10,7 +10,7 @@ interface BrowserWallet extends AppWallet {
 }
 
 export const useBrowserWallet: () => BrowserWallet = () => {
-  const { wallet }: { wallet: WalletLocked } = useWallet();
+  const { wallet }: { wallet: Account | null | undefined } = useWallet();
```

Comment:
Same here

Reply:
I believe Wallet is derived from Account.

Reply:
Understood, perhaps it should be optional then? Having a type that could also be `null` or `undefined` weakens the benefits of idiomatic typescript. I would say that [useWallet](https://github.com/FuelLabs/fuels-npm-packs/blob/main/packages/react/src/hooks/useWallet.ts?rgh-link-date=2024-04-17T08%3A17%3A14Z) should have a return type as well.